features/support/application_cuke_helpers.rb

```diff
@@ -32,6 +32,9 @@ def check_fields_validation_error(field_list)
     end
   end
 
+  def mobile_version?
```

Comment:
There is probably a cleaner way to know in the cukes if we are using the mobile version,